modules/core/pom.xml

```diff
@@ -141,6 +141,33 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <phase>package</phase>
+            <configuration>
+              <minimizeJar>true</minimizeJar>
+              <artifactSet>
+                <includes>
+                  <include>org.apache.thrift:libthrift</include>
```

Comment:
Perhaps the original issue should have been more specific, but the intent was to [relocate][1] the libthrift classes. If we don't do the relocation, then the original conflict with Accumulo's libthrift dependency will not be resolved.

[1]: https://maven.apache.org/plugins/maven-shade-plugin/examples/class-relocation.html